submissions/ses3332002/memory-game/script.js

```diff
@@ -0,0 +1,156 @@
+"use strict";
+  
+  const doc = document.body;
+  const themeButtons = document.querySelectorAll(".theme_selector__button");
+  const sizeSelector = document.querySelector(".size_selector");
+  const cardThemeSelector = document.querySelectorAll('input[name="option"]');
+  const startButton = document.querySelector(".start_button");
+  const gameArea = document.querySelector(".game_area");
+  let numbersArray;
+  let cardThemeSelected;
+  let cardSelected;
+  let cardsArray;
+  let matchedCounter;
+  
+  startButton.addEventListener("click", startGame);
+  doc.classList.add("theme1");
+
+  themeButtons.forEach((el,i) => {
+    el.classList.add("theme"+(i+1));
+    el.addEventListener("click", function() {
+      setupTheme(i);
+    });
+  });
+
+  function setupTheme(i) {
+    doc.classList.remove(doc.classList[0]);
+    doc.classList.add("theme"+(i+1));
+  };
+  
+  function startGame() {
+    cardThemeSelected = document.querySelector('input[name="option"]:checked');
+    cardThemeSelector.forEach((item) => {
+      item.disabled = true;
+      item.nextElementSibling.classList.add("form_item-desabled");
+      item.nextElementSibling.style.pointerEvents = "none";
```

Comment:
The property "pointer-events: none;" of course can be written in class "form_item-disabled", I don't know why I didn't notice it myself :)
But "item.disabled = true;" must be left, because without it controls are still accessible via keyboard.

Reply:
Quite agree 🙂

Reply:
I remade controls switching